patches/server/0873-Fix-a-bunch-of-vanilla-bugs.patch

```diff
@@ -48,6 +48,9 @@ https://bugs.mojang.com/browse/MC-123848
   by: BillyGalbreath <blake.galbreath@gmail.com>
   Fixes item frames dropping items above when pointing down
 
+https://bugs.mojang.com/browse/MC-264274
```

Comment:
This is not the right fix for this bug

Reply:
True lol, machine copying the wrong mojira id smh